public/src/main/java/com/revenuecat/purchases/models/GoogleSubscriptionOption.kt

```diff
@@ -12,8 +12,22 @@ data class GoogleSubscriptionOption(
     override val pricingPhases: List<PricingPhase>,
     override val tags: List<String>,
     val productDetails: ProductDetails,
-    val offerToken: String
+    val offerToken: String,
+    override val presentedOfferingIdentifier: String? = null
```

Comment:
Maybe for another PR, but since this class is public, we should add docs for it as well